src/pages/Gallery/index.tsx

```diff
@@ -1,5 +1,59 @@
-const GalleryPage = () => {
-  return <h1>GalleryPage</h1>
+import * as React from 'react'
+import * as S from './Gallery.styles'
+
+import MenuBar from './components/Category'
+import List from './components/PictureList'
+
+type Picture = {
+  imageUrl: string
+  title: string
+  links: { href: string }[]
+}
+
+const GalleryPage: React.FC = () => {
+  const [pictures, setPictures] = React.useState<Picture[]>([])
+  const [selectedCategory, setSelectedCategory] = React.useState<string>('행성')
```

Comment:
여기서 타입으로 
```suggestion
  const [selectedCategory, setSelectedCategory] = useState<{'행성' | '우주' | '발사체' | '인공위성'}>('행성')
```
위처럼 들어올 수 있는 타입을 한정해주는 것도 좋을 거 같다는 생각임미다